wiki/Tournaments/Official_support/en.md

```diff
@@ -25,12 +25,17 @@ Officially supported tournaments have access to:
 
 Community-run tournaments which abide by the following hard criteria are eligible for support:
 
-- The tournament series must run no more than four times per year.
-- The tournament must operate on a Round of 16 double-elimination format at a minimum, **OR**, have at least 64 players (or 16 teams) participating. Larger scales and formats are permitted.
-- Staff members must not participate in the tournament themselves at any point, and players are unable to later assist as staff either. For our definition on "staff", please refer to the [FAQ](#what-is-considered-staff).
+- The tournament series must run no more than two times per year.
+- The tournament must operate on a Round of 16 double-elimination format at a minimum — alternative scales and formats may be permitted at the sole discretion of the osu! team. Hosts should contact [tournaments@ppy.sh](mailto:tournaments@ppy.sh) if they are seeking to receive tournament support for a non-traditional tournament format or size.
 - The forum thread for the tournament **MUST** contain a clearly visible link in a normal-sized font to the [tournament reports form](https://pif.ephemeral.ink/tournament-reports) as the very last content of the original post.
   - `https://pif.ephemeral.ink/tournament-reports`
   - This report form is overseen by the [Tournament Committee](/wiki/People/Tournament_Committee). We encourage all users — players and staff alike — to make use of this form where necessary. Any breaches of [expectations](#expectations), [other procedures](#other-procedures), or [eligibility](#eligibility) requirements should prompt a report.
+- The tournament only allows players of exceptional skill to participate, disallowing players lower than the following ranks in each mode to compete with a chance of winning the tournament:
+  - osu!: 100,000
+  - osu!taiko: 10,000
+  - osu!catch: 5,000
+  - osu!mania: 40,000
```

Comment:
what are these based on? some percentile of the playerbase?

Reply:
The numbers here were decided and agreed upon during an official Tournament Committee meeting.

Reply:
bit random but ok